.github/workflows/main.yml

```diff
@@ -38,6 +38,9 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-prebuild-
       - run: ./gimme module | grep ^export | tee -a "${GITHUB_ENV}"
+      - uses: actions/setup-go@v5
```

Comment:
This feels moderately ironic, given the purpose of this repository/tool :joy:

Reply:
It amused me too. 😆